js/libs/build.js

```diff
@@ -378,6 +378,12 @@ Fliplet.Widget.instance('interactive-map', function(widgetData) {
         await this.init()
       }
 
+      Fliplet.Hooks.on('appearanceChanged', () => {
+        // The PanZoom library is listening to the window resize
+        // to calculate the new position of the map
+        $(window).resize()
```

Comment:
The PanZoom library [has a line](https://github.com/Fliplet/fliplet-api/blob/master/public/assets/fliplet-ui-panzoom/1.0/panzoom.js#L1003) as shown below:

```js
$(window).resize(resetElement);
```

Can the `resetElement()` function be exposed for the `Fliplet.UI.PanZoom` namespace so that you can just run:

```suggestion
        Fliplet.UI.PanZoom.whateverFunctionName();
```

Reply:
Sure! Let me update that!

Reply:
Excellent. We should avoid triggering the window resize, which was the whole point of all this.